src/components/FollowComponent/singleUserFollow.js

```diff
@@ -0,0 +1,60 @@
+import {Avatar, Button, GridItem, Text, SimpleGrid} from "@chakra-ui/react";
+import {useNavigate} from "react-router";
+import {useDispatch, useSelector} from "react-redux";
+import {updateUser} from "../../actions/user-actions";
+import {loginUser} from "../../actions/current-user-actions";
+
+
```

Comment:
Here's the logic that adds to each other's following/followers lists and updates the users